public-ui/src/app/data-browser/views/quick-search/home-view-react.component.tsx

```diff
@@ -66,19 +70,28 @@ export const dBHomeComponent = (
             this.state = {
                 surveyInfo: [],
                 domainInfo: [],
-                physicalMesurmentsInfo: []
+                physicalMesurmentsInfo: [],
+                searchWord: ''
             };
         }
 
+        search = _.debounce((val) => {
+            this.setState({searchWord: val});
+            this.getDomainInfos();
+            },
+        1000);
```

Comment:
If dropped state from `SearchComponent`, need to update `searchWord` outside the debounce:
```suggestion
        search = _.debounce((val) => this.getDomainInfos(), 1000);
```

Reply:
Changed this. Thanks.

Reply:
Changed this. Thanks.